tests/start_proxy/start_proxy_test.py

```diff
@@ -138,6 +138,24 @@ def test_gen_proxy_config(self):
               '--listener_port', '8080', '--ssl_client_cert_path',
```

Comment:
add conflict test of enable_grpc_backend_ssl and ssl_client_root_certs_file

Reply:
already done in line 342